tests/test_vault.yml

```diff
@@ -28,4 +28,42 @@
       import_role:
         name: vault_unseal
       vars:
-        vault_keys: "{{ vault_keys.keys_base64 }}"
+        vault_unseal_keys: "{{ vault_keys.keys_base64 }}"
+
+    - name: Configure PKI
```

Comment:
We could followup with exporting the intermediate here, scratch the vault, redeploy, and test the intermediate import functionality.